src/EventRegistrar.php

```diff
@@ -257,14 +256,13 @@ protected static function handleUpdateEvent(Event $event)
             Plugin::getInstance()->trigger(Plugin::EVENT_BEFORE_PURGE, $purgeEvent);
 
             // Push to queue
-            \Craft::$app->getQueue()->push(new PurgeCacheJob([
+            \Craft::$app->getQueue()->ttr(1200)->push(new PurgeCacheJob(
```

Comment:
Any good reason for `ttr(1200)`?

Reply:
In our experience, with sites that have a lot of URLs the purge by url loop takes too long and times out with the default TTR. We actually purge by tag now to avoid this problem, so I guess the ttr might indeed be overkill in most cases.